src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(operand) {
+      this.result += operand;
+
+      return this;
+    },
+    subtract(operand) {
```

Comment:
Consider will be better if your fix it everywhere
```suggestion
    subtract(value) {
```